scripts/test/compactionPerformance/system-test-instance.properties

```diff
@@ -65,162 +65,11 @@ sleeper.optional.stacks=SystemTestStack,CompactionStack,IngestStack
-sleeper.gc.batch.size=2000
-
-# The length of time after a message is consumed from a queue before it becomes visible again
-sleeper.queue.visibility.timeout.seconds=900
-
-# The frequency with which a compaction job sends keep alive messages to the job queue
-sleeper.compaction.keepalive.period.seconds=600
-
-# The rate at which the compaction job creation lambda runs (in minutes, must be >=1)
-sleeper.compaction.job.creation.period.minutes=1
-
-# The rate at which a check to see if compaction ECS tasks need to be created is made (in minutes, must be >= 1)
-sleeper.compaction.task.creation.period.minutes=1
-
-# The amount of memory for the lambda that creates compaction jobs
-sleeper.compaction.job.creation.memory=1024
-
-# The amount of memory for the lambda that runs ECS tasks to execute compaction jobs
-sleeper.task.runner.memory=1024
-
-# The timeout for the lambda that creates compaction jobs in seconds
-sleeper.compaction.job.creation.timeout.seconds=900
-
-# The timeout for the lambda that creates compaction jobs in seconds
-sleeper.task.runner.timeout.seconds=900
-
-# The CPU architecture to run compaction tasks on.
-# See Task CPU architecture at https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html
-sleeper.compaction.task.cpu.architecture=X86_64
-
-# The CPU and memory for a compaction task. Which of these are used will depend on the chosen CPU architecture.
-# See https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html for valid options.
-sleeper.compaction.task.arm.cpu=1024
-sleeper.compaction.task.arm.memory=4096
-sleeper.compaction.task.x86.cpu=1024
-sleeper.compaction.task.x86.memory=4096
-
-# Flag to enable/disable storage of tracking information for compaction jobs and tasks.
-sleeper.compaction.status.store.enabled=true
-
-# The time to live in seconds for compaction job updates in the status store. Default is 1 week
-sleeper.compaction.job.status.ttl=604800
-
-# The time to live in seconds for compaction task updates in the status store. Default is 1 week
-sleeper.compaction.task.status.ttl=604800
-
-# The retention period for CloudWatch logs. See https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-loggroup.html for valid options.
-# Use -1 to indicate infinite retention.
-sleeper.log.retention.days=30
-
-# The amount of memory in MB for the lambda that identifies partitions that need splitting
-sleeper.partition.splitting.finder.memory=2048
-
-# The timeout for the lambda that identifies partitions that need splitting in seconds
-sleeper.partition.splitting.finder.timeout.seconds=900
-
-# The amount of memory for the lambda that splits partitions
-sleeper.partition.splitting.memory=2048
-
-# The timeout for the lambda that splits partitions in seconds
-sleeper.partition.splitting.timeout.seconds=900
-
-# The maximum number of simultaneous connections to S3 from a single client
-sleeper.s3.max-connections=25
```

Comment:
The max connections may be relevant to the performance of compaction and ingest.